core/commands/repo.go

```diff
@@ -39,6 +40,11 @@ var RepoCmd = &cmds.Command{
 	},
 }
 
+type GcResult struct {
```

Comment:
Every type exported and every attributes that's also exported should have a well described comment above.